Automattic/jetpack
# Publicize: handle main toggle control

This PR changes a little bit the way that the main Disable control is handled by the feature.
Now, the toggle control shows only when the post has not been published yet. Also, we changed the label with `Share when publishing`.

Fixes https://github.com/Automattic/jetpack/issues/21525

#### Changes proposed in this Pull Request:

* Publicize: handle main toggle control

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Enable the RePublicize feature ([issue with instructions](https://github.com/Automattic/jetpack/pull/21366)).
* Create a new post
* Confirm you can see the main toggle control
<img src="https://user-images.githubusercontent.com/77539/138877086-e7b5b0d3-4e41-4fab-8e6b-31336f07833c.png" width="300px" />

* Once the post is published, confirm that the toggle control is not there anymore for sites with plans that support the `republicize` feature
<img src="https://user-images.githubusercontent.com/77539/138877360-e2e13c02-c3f1-479a-afed-d1180408e68f.png" width="300px" />

* Confirm that you get an upsell notice for sites that don't support the feature.
 
<img src="https://user-images.githubusercontent.com/77539/138877468-b24dd95b-ad55-4f57-9382-4dfc6e46729b.png" width="300px" />
